src/sentry/api/endpoints/project_rules.py

```diff
@@ -73,24 +82,363 @@ def find_duplicate_rule(rule_data, project, rule_id=None):
+- `match`: The comparison operator. Valid values are `eq` (equals), `ne` (does not equal), `sw` (starts with), `ew` (ends with), `co` (contains), `nc` (does not contain), `is` (is set), and `ns` (is not set).
+- `value`: A string. Not required when `match` is `is` or `ns`.
+```json
+{
+    "id": "sentry.rules.conditions.event_attribute.EventAttributeCondition",
+    "attribute": "http.url",
+    "match": "nc",
+    "value": "localhost"
+}
+```
+
+**The event's tags match `key` `match` `value`**
+- `key`: The tag
+- `match`: The comparison operator. Valid values are `eq` (equals), `ne` (does not equal), `sw` (starts with), `ew` (ends with), `co` (contains), `nc` (does not contain), `is` (is set), and `ns` (is not set).
+- `value`: A string. Not required when `match` is `is` or `ns`.
+```json
+{
+    "id": "sentry.rules.filters.tagged_event.TaggedEventFilter",
+    "key": "level",
+    "match": "eq"
+    "value": "error"
+}
+```
+
+**The event's level is `match` `level`**
+- `match`: Valid values are `eq`, `gte`, and `lte`.
+- `level`: Valid values are `50` (fatal), `40` (error), `30` (warning), `20` (info), `10` (debug), `0` (sample).
+```json
+{
+    "id": "sentry.rules.filters.level.LevelFilter",
+    "match": "gte"
+    "level": "50"
+}
+```
+""",
+    )
+    owner = ActorField(
+        required=False, allow_null=True, help_text="The ID of the team or user that owns the rule."
+    )
+
+
+@extend_schema(tags=["Events"])
 @region_silo_endpoint
 class ProjectRulesEndpoint(ProjectEndpoint):
     publish_status = {
-        "GET": ApiPublishStatus.UNKNOWN,
-        "POST": ApiPublishStatus.UNKNOWN,
+        "GET": ApiPublishStatus.PUBLIC,
+        "POST": ApiPublishStatus.PUBLIC,
     }
     owner = ApiOwner.ISSUES
     permission_classes = (ProjectAlertRulePermission,)
 
+    @extend_schema(
+        operation_id="List a Project's Issue Alert Rules",
+        parameters=[GlobalParams.ORG_SLUG, GlobalParams.PROJECT_SLUG],
+        request=None,
+        responses={
+            200: inline_sentry_response_serializer("ListRules", List[RuleSerializerResponse]),
+            401: RESPONSE_UNAUTHORIZED,
+            403: RESPONSE_FORBIDDEN,
+            404: RESPONSE_NOT_FOUND,
+        },
+        examples=IssueAlertExamples.LIST_PROJECT_RULES,
+    )
     @transaction_start("ProjectRulesEndpoint")
     def get(self, request: Request, project) -> Response:
         """
-        List a project's rules
-
-        Retrieve a list of rules for a given project.
-
-            {method} {path}
+        Return a list of active issue alert rules bound to a project.
 
+        An issue alert rule triggers whenever a new event is received for any issue in a project that matches the specified criteria. These criteria can include a resolved issue re-appearing or an issue affecting many users. In issue alert rules, Sentry evaluates the configured alert conditions each time it receives a new event. Alert conditions have three parts:
```

Comment:
An optional suggestion to make this shorter & clearer:

```suggestion
        An issue alert rule triggers whenever a new event is received for any issue in a project that matches the specified alert conditions. These conditions can include a resolved issue re-appearing or an issue affecting many users. Alert conditions have three parts:
```